dispatcher/src/main/java/io/zeebe/dispatcher/Dispatcher.java

```diff
@@ -142,6 +145,7 @@ public long claim(final ClaimedFragment claim, final int length, final int strea
         (partition, activePartitionId) ->
             logAppender.claim(
                 partition, activePartitionId, claim, length, streamId, onClaimComplete),
+        1,
```

Comment:
The offer method takes the fragment count. A batch fragment can have multiple fragments but this method is for a single fragment so the fragment count is 1. I can add a `claimedFragment.getCount()` but it seems silly since it can only be 1.

Reply:
maybe `logAppender.claimSingleFragment()` for this case to make it a bit clearer.